### PR TITLE
enable-api does not need a model.

### PR DESCRIPTION
**Pull request by tabergma**
**Proposed changes**:
`rasa run --enable-api` does not need a model. The server can start without any model as you are able to train a model via the API once the server is up and running.

closes https://github.com/RasaHQ/rasa/issues/3588

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

File: rasa/cli/run.py [hunk 1]
```diff
@@ -60,6 +60,20 @@ def run(args: argparse.Namespace):
     import rasa.run
 
     args.model = get_validated_path(args.model, "model", DEFAULT_MODELS_PATH)
```
Comment (wochinge) [inline rasa/cli/run.py]: I think we should move this line in the `if not args.enable_api` block and set it to `None` or so.

Comment (wochinge) [reply rasa/cli/run.py]: Cause atm if fails there is no directory `models` and you try to run it with `--enable-api`

Comment (tabergma) [reply rasa/cli/run.py]: We can move it insight, but then the following might happen:
You run `rasa run --enable-api -m non-existing-model`. We are not validating if `non-existing-model` is a valid path. Thus the output will look like:
``` 
No chat connector configured, falling back to the REST input channel. To connect your bot to another channel, read the docs here: https://rasa.com/docs/rasa/user-guide/messaging-and-voice-channels
2019-06-05 11:27:49 INFO     root  - Starting Rasa Core server on http://localhost:5005
2019-06-05 11:27:49 WARNING  rasa.core.agent  - No valid configuration given to load agent.
2019-06-05 11:27:49 WARNING  root  - Agent could not be loaded with the provided configuration. Load default agent without any model.
```
Not sure if that is clear to the user.

Comment (wochinge) [reply rasa/cli/run.py]: Good point. Should we try to create the directory if it does not exist?

Comment (tabergma) [reply rasa/cli/run.py]: Updated the PR.

**Pull request merged**